### ISSUE TITLE
Documentation Not  Clear Regarding node oAuth

### ISSUE THREAD

**Issue by siddmegadeth** [ciaranj/node-oauth#273]
what is  'your user toke for this app', //test user token
  'your user secret for this app', //test user secret  . Is it the consumer key from twtter or any string will do

**Comment by omouse** [comment]
Agreed, the docs are pretty bad, they're missing the fact that you need to get the oauth authorization before getting the access token.
